### PR TITLE
Remove code for rendering `/courts-tribunals`

### PR DESCRIPTION
**Pull request by brucebolt**
This removes the code used to render [`/courts-tribunals`](/courts-tribunals).  The route has been redirected to [`/government/organisations/hm-courts-and-tribunals-service/about`](/government/organisations/hm-courts-and-tribunals-service/about), so we no longer need the code to render the content in Whitehall.

Tidying of the leftover routes (i.e. creating proper route helpers for the redundant paths) will be done as part of a separate piece of work.

[Trello card](https://trello.com/c/LgmL6WvW/85-put-a-redirect-on-courts-and-tribunals)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -408,7 +408,7 @@ def external_redirect(path_prefix, target)
     get "/placeholder" => "placeholder#show", as: :placeholder
   end
 
-  get "/courts-tribunals(.:locale)", as: "courts", to: "organisations#index", courts_only: true, constraints: { locale: valid_locales_regex }
+  # TODO: the organisations controller has been removed but this route is still required to get the relevant helper methods. This can be removed once new helpers have been created.
```
Comment (BeckaL) [inline config/routes.rb:411]: Not sure I fully understand why we still need this, but should we fix this TODO as part of this PR?

Comment (brucebolt) [reply config/routes.rb:411]: This is a longstanding issue when rendering has been migrated out of Whitehall, as other parts of Whitehall use the path helpers that Rails generates when routes exist (e.g. `organisations_path` in this case).

There are multiple other examples, e.g. [for get-involved](https://github.com/alphagov/whitehall/blob/42d3182d97ff2d28e6140c4f6eabc09f527620e6/config/routes.rb#L68-L72), so we have a [separate Trello card](https://trello.com/c/EfhUs8ak/33-tidy-whitehalls-routes-file) to sort out this issue by creating our own helper methods and then removing them from the routes file.

**Review by BeckaL — APPROVED** [review]
Looks good. And a satisfying tidying up of whitehall - that negative line count 🤩

**Review by JonathanHallam — APPROVED** [review]
This is an awesome chunk of code removal! Thanks Bruce

**Pull request merged**